packages/paste-design-tokens/formatters/sketchpalette.ts

```diff
@@ -17,6 +17,7 @@ interface ColorRGB {
 }
 
 const convertToSketchPaletteColor = (input: string): {[key: string]: number} => {
+  // @ts-ignore FIXME
```

Comment:
We should fix this, the typing is way off.  Didn't want it to block the PR though.